From 50472ab8ccdc14a16b4d4a59a7a695f226fb7d80 Mon Sep 17 00:00:00 2001
From: "Daniel M. Capella" <polyzen@archlinux.org>
Date: Sun, 3 Nov 2024 19:09:45 -0500
Subject: [PATCH] Comment out encrypted credentials

Make this opt-in. The service errors out if credentials are not found.
---
 sample/systemd/borgmatic.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sample/systemd/borgmatic.service b/sample/systemd/borgmatic.service
index a3391c32..e5092646 100644
--- a/sample/systemd/borgmatic.service
+++ b/sample/systemd/borgmatic.service
@@ -11,8 +11,8 @@ Documentation=https://torsion.org/borgmatic/
 Type=oneshot
 
 # Load encrypted credentials.
-LoadCredentialEncrypted=borgmatic:/etc/credstore.encrypted/borgmatic/
-LoadCredentialEncrypted=borgmatic.pw
+# LoadCredentialEncrypted=borgmatic:/etc/credstore.encrypted/borgmatic/
+# LoadCredentialEncrypted=borgmatic.pw
 
 # Security settings for systemd running as root, optional but recommended to improve security. You
 # can disable individual settings if they cause problems for your use case. For more details, see
-- 
2.47.0

